tests/ci/build_check.py

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env python3
 
+from typing import List, Tuple
 import subprocess
 import logging
 import json
 import os
 import sys
 import time
-from typing import List, Tuple
+import urllib.parse
+import requests
```

Comment:
I think, the goal is to use the same DB as for checks, so https://github.com/ClickHouse/ClickHouse/blob/master/tests/ci/clickhouse_helper.py#L18 would be extended

Reply:
No, we will use a separate DB because the DB with checks is capped at about 3.6 TB, while ClickHouse Cloud is unlimited.